docs/examples/notebooks/learn/TestStatistics.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>Make a callback to the fact that the toys were generated with <code>\mu' = 1</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scikit-hep/pyhf/pull/1014/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
they both use now a common variable